### DISCUSSION TITLE
建议边缘滚动功能中右边的亮度调节设置为系统内置的亮度调节！

### DISCUSSION THREAD

**Discussion by nigeder** [shuax/MouseInc.Settings#73] [Ideas]
希望能像左边的音量调节功能一样调用系统内置键值对进行亮度的调节，目前弄的那个因为和系统不同步所以每次重启都没有记忆功能，而且容易受其他护眼软件的影响，在其他软件的调节亮度下，软件所做的亮度调节功能会完全失效，所以强烈建议能想办法让右侧的亮度调节功能向左侧的声音调节功能那样调用系统操作！之前尝试过自己定义调节的快捷键但是失败了，所以希望能更新系统调节亮度的功能，或者告诉我设置的代码也行
![image](https://user-images.githubusercontent.com/66513464/126164581-dc0bdc9c-25c7-49dc-a475-28c1027130ba.png)

**Comment by shuax** [comment]
我研究一下，调节亮度比调节音量复杂得多。

**Comment by shuax** [comment]
已经调整

**Reply by nigeder** [reply]
看到了你最近的更新了，确实优化了不少东西，但是其实我之前之所以建议最好调用系统的亮度调节是因为系统的亮度调节和音量调节一样是有记忆功能的，不会因为重启而恢复默认亮度，而这点是软件目前尚未解决的问题，就是说在用户设定一个亮度之后，能否让软件重启、开机重启之后只要开启软件，依然恢复到该亮度。

**Reply by leimuhhh** [reply]
更新完我现在连亮度都调不了了
